src/plugins/rangePlugin.ts

```diff
@@ -178,10 +178,9 @@ function rangePlugin(config: Config = {}): Plugin {
           _prevDates = [...newDates];
         }
 
-        [
-          fp._input.value = "",
-          secondInput.value = "",
-        ] = fp.selectedDates.map(d => fp.formatDate(d, dateFormat));
+        [fp._input.value = "", secondInput.value = ""] = fp.selectedDates.map(
```

Comment:
Shouldn't we ignore eslint for those file to more easily find our changes ?

Reply:
We could, but since it's linted now, we will be good :)